.github/workflows/codecov.yaml

```diff
@@ -0,0 +1,18 @@
+name: Codecov - upload coverage report
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Test & coverage
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup Python
+      uses: actions/setup-python@master
+      with:
+        python-version: 3.6
+    - name: Generate coverage report
+      run: |
+        pip install pytest-cov
+        pytest --cov --cov-report xml .
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v2
```

Comment:
Actually! for the sake of consistency, as I don't want to do yet another CI migration without a good reason... could you use CircleCI's `codecov/codecov` Orb and plug it into the existing pipeline in `.circleci/config.yml`? There's a ready sample in our another open source project, [VirtusLab-Open-Source/strapi-plugin-comments](https://github.com/VirtusLab-Open-Source/strapi-plugin-comments/blob/master/.circleci/config.yml)

Reply:
This will also make it easier to actually test the coverage upload without setting up GH Actions on this repo 😅

Reply:
To stay within the existing tooling, pls add `coverage xml` to `testenv:coverage`->`commands` within tox.ini; then you'll be able to simply upload the generated `coverage.xml` file to Codecov as a `codecov/upload` `step` under `test against python 3_9 and git 2_30_1 with coverage report` job in CircleCI config

Reply:
No problem. I'll add Cidecov integration to CicrleCi pipeline.

Reply:
DONE!!!
GitHub Action - removed
CircleCi - updated

Next steps: 
- [Getting a Codecov account](https://docs.codecov.com/docs/step-2-getting-a-codecov-account-and-uploading-coverage)
- fix [Codecov badge](https://docs.codecov.com/docs/status-badges)

Reply:
Gr8 🎉 lemme see codecov.com now 🕵🏻

Reply:
Seems to work:

https://app.codecov.io/gh/VirtusLab/git-machete/commits?page=1

Reply:
> fix [Codecov badge](https://docs.codecov.com/docs/status-badges)

Badge doesn't need fixing, just a matter of getting these changes merged to master (will happen with the next release soon). Then the badge will show up in the main repo page's README, with correct percentage

Also, looks that the token attached to the codecov.io URL is redundant now